diff --git a/src/Controller/AnswerController.php b/src/Controller/AnswerController.php
index 7295caea..53a3c301 100644
--- a/src/Controller/AnswerController.php
+++ b/src/Controller/AnswerController.php
@@ -2,6 +2,8 @@
 
 namespace App\Controller;
 
+use App\Entity\Answer;
+use Doctrine\ORM\EntityManagerInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
@@ -12,22 +14,23 @@ class AnswerController extends AbstractController
     /**
      * @Route("/answers/{id}/vote", methods="POST", name="answer_vote")
      */
-    public function answerVote($id, LoggerInterface $logger, Request $request)
+    public function answerVote(Answer $answer, LoggerInterface $logger, Request $request, EntityManagerInterface $entityManager)
     {
         $data = json_decode($request->getContent(), true);
         $direction = $data['direction'] ?? 'up';
 
-        // todo - use id to query the database
-
         // use real logic here to save this to the database
         if ($direction === 'up') {
             $logger->info('Voting up!');
+            $answer->setVotes($answer->getVotes() + 1);
             $currentVoteCount = rand(7, 100);
         } else {
             $logger->info('Voting down!');
-            $currentVoteCount = rand(0, 5);
+            $answer->setVotes($answer->getVotes() - 1);
         }
 
-        return $this->json(['votes' => $currentVoteCount]);
+        $entityManager->flush();
+
+        return $this->json(['votes' => $answer->getVotes()]);
     }
 }
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 17e417ce..15ba9462 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -68,7 +68,7 @@
                             class="vote-arrows"
                             {{ stimulus_controller('answer-vote', {
                                 url: path('answer_vote', {
-                                    id: 10
+                                    id: answer.id
                                 })
                             }) }}
                         >
