diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 150aeda..f998843 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -16,7 +16,7 @@
                                 <small>
                                     {% if question.askedAt %}
                                         Asked <br>
-                                        {{ question.askedAt|date('Y-m-d H:i:s') }}
+                                        {{ question.askedAt|ago }}
                                     {% else %}
                                         (unpublished)
                                     {% endif %}
