diff --git a/public/js/question_show.js b/public/js/question_show.js
new file mode 100644
index 0000000..7332aac
--- /dev/null
+++ b/public/js/question_show.js
@@ -0,0 +1,15 @@
+/**
+ * Simple (ugly) code to handle the comment vote up/down
+ */
+var $container = $('.js-vote-arrows');
+$container.find('a').on('click', function(e) {
+    e.preventDefault();
+    var $link = $(e.currentTarget);
+
+    $.ajax({
+        url: '/comments/10/vote/'+$link.data('direction'),
+        method: 'POST'
+    }).then(function(response) {
+        $container.find('.js-vote-total').text(response.votes);
+    });
+});
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 12e1f4d..c7440ee 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -56,3 +56,9 @@
     </ul>
 </div>
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+
+    <script src="{{ asset('js/question_show.js') }}"></script>
+{% endblock %}
