diff --git a/src/Controller/CommentController.php b/src/Controller/CommentController.php
index d7b111a..c94477d 100644
--- a/src/Controller/CommentController.php
+++ b/src/Controller/CommentController.php
@@ -22,6 +22,6 @@ class CommentController extends AbstractController
             $currentVoteCount = rand(0, 5);
         }
 
-        return new JsonResponse(['votes' => $currentVoteCount]);
+        return $this->json(['votes' => $currentVoteCount]);
     }
 }
