diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 65bd3364..12e1f4da 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -45,10 +45,10 @@
                         {{ answer }}
                         <p>-- Mallory</p>
                     </div>
-                    <div class="vote-arrows flex-fill pt-2" style="min-width: 90px;">
-                        <a class="vote-up" href="#"><i class="far fa-arrow-alt-circle-up"></i></a>
-                        <a class="vote-down" href="#"><i class="far fa-arrow-alt-circle-down"></i></a>
-                        <span>+ 6</span>
+                    <div class="vote-arrows flex-fill pt-2 js-vote-arrows" style="min-width: 90px;">
+                        <a class="vote-up" href="#" data-direction="up"><i class="far fa-arrow-alt-circle-up"></i></a>
+                        <a class="vote-down" href="#" data-direction="down"><i class="far fa-arrow-alt-circle-down"></i></a>
+                        <span>+ <span class="js-vote-total">6</span></span>
                     </div>
                 </div>
             </li>
