diff --git a/templates/base.html.twig b/templates/base.html.twig
index 4f1651e..cb783df 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -23,6 +23,11 @@
         <footer class="mt-5 p-3 text-center">
             Made with <i style="color: red;" class="fa fa-heart"></i> by the guys and gals at <a style="color: #444; text-decoration: underline;" href="https://symfonycasts.com">SymfonyCasts</a>
         </footer>
-        {% block javascripts %}{% endblock %}
+        {% block javascripts %}
+            <script
+              src="https://code.jquery.com/jquery-3.4.1.min.js"
+              integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
+              crossorigin="anonymous"></script>
+        {% endblock %}
     </body>
 </html>
