diff --git a/src/Controller/UserController.php b/src/Controller/UserController.php
new file mode 100644
index 0000000..a5b6cfa
--- /dev/null
+++ b/src/Controller/UserController.php
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\Routing\Annotation\Route;
+
+class UserController extends BaseController
+{
+    /**
+     * @Route("/api/me")
+     * @IsGranted("IS_AUTHENTICATED_REMEMBERED")
+     */
+    public function apiMe()
+    {
+        return $this->json($this->getUser());
+    }
+}
