diff --git a/src/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
index 54f44f2c..4191c5ea 100644
--- a/src/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -13,12 +13,12 @@ class LoginFormAuthenticator extends AbstractAuthenticator
 {
     public function supports(Request $request): ?bool
     {
-        dd('supports!');
+        return ($request->getPathInfo() === '/login' && $request->isMethod('POST'));
     }
 
     public function authenticate(Request $request): PassportInterface
     {
-        // TODO: Implement authenticate() method.
+        dd('authenticate!');
     }
 
     public function onAuthenticationSuccess(Request $request, TokenInterface $token, string $firewallName): ?Response
