diff --git a/src/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
index c5d33a1e..ee36fdb0 100644
--- a/src/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -46,19 +46,19 @@ class LoginFormAuthenticator extends AbstractAuthenticator
                 return $user;
             }),
             new CustomCredentials(function($credentials, User $user) {
-                dd($credentials, $user);
+                return $credentials === 'tada';
             }, $password)
         );
     }
 
     public function onAuthenticationSuccess(Request $request, TokenInterface $token, string $firewallName): ?Response
     {
-        // TODO: Implement onAuthenticationSuccess() method.
+        dd('success');
     }
 
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception): ?Response
     {
-        // TODO: Implement onAuthenticationFailure() method.
+        dd('failure');
     }
 
 //    public function start(Request $request, AuthenticationException $authException = null): Response
