diff --git a/src/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
index df465c5d..1c9efffc 100644
--- a/src/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -33,12 +33,6 @@ class LoginFormAuthenticator extends AbstractLoginFormAuthenticator
     {
     }
 
-    public function supports(Request $request): bool
-    {
-        return self::LOGIN_ROUTE === $request->attributes->get('_route')
-            && $request->isMethod('POST');
-    }
-
     public function authenticate(Request $request)
     {
         // TODO: Implement authenticate() method.
