diff --git a/config/packages/stof_doctrine_extensions.yaml b/config/packages/stof_doctrine_extensions.yaml
index c83f3b1..e7ac8f4 100644
--- a/config/packages/stof_doctrine_extensions.yaml
+++ b/config/packages/stof_doctrine_extensions.yaml
@@ -2,3 +2,6 @@
 # See the official DoctrineExtensions documentation for more details: https://github.com/Atlantic18/DoctrineExtensions/tree/master/doc/
 stof_doctrine_extensions:
     default_locale: en_US
+    orm:
+        default:
+            sluggable: true
diff --git a/src/Factory/QuestionFactory.php b/src/Factory/QuestionFactory.php
index a4a0ca4..8687a11 100644
--- a/src/Factory/QuestionFactory.php
+++ b/src/Factory/QuestionFactory.php
@@ -42,12 +42,7 @@ final class QuestionFactory extends ModelFactory
     {
         // see https://github.com/zenstruck/foundry#initialization
         return $this
-            ->afterInstantiate(function(Question $question) {
-                if (!$question->getSlug()) {
-                    $slugger = new AsciiSlugger();
-                    $question->setSlug($slugger->slug($question->getName()));
-                }
-            })
+            //->afterInstantiate(function(Question $question) { });
         ;
     }
 
