diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index c948d6e3..0bd44e04 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -5,12 +5,15 @@ namespace App\Entity;
 use App\Repository\QuestionRepository;
 use Doctrine\ORM\Mapping as ORM;
 use Gedmo\Mapping\Annotation as Gedmo;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
 
 /**
  * @ORM\Entity(repositoryClass=QuestionRepository::class)
  */
 class Question
 {
+    use TimestampableEntity;
+
     /**
      * @ORM\Id()
      * @ORM\GeneratedValue()
