diff --git a/src/Entity/Comment.php b/src/Entity/Comment.php
index b8c024c3..3e7491a7 100644
--- a/src/Entity/Comment.php
+++ b/src/Entity/Comment.php
@@ -4,12 +4,15 @@ namespace App\Entity;
 
 use App\Repository\CommentRepository;
 use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Timestampable\Traits\TimestampableEntity;
 
 /**
  * @ORM\Entity(repositoryClass=CommentRepository::class)
  */
 class Comment
 {
+    use TimestampableEntity;
+
     /**
      * @ORM\Id
      * @ORM\GeneratedValue
