diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 8d54821e..3244aa15 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -18,13 +18,12 @@ class AppFixtures extends Fixture
     {
         TagFactory::createMany(100);
 
-        $questions = QuestionFactory::createMany(20, function() {
+        $questions = QuestionFactory::createMany(20);
+
+        QuestionTagFactory::createMany(100, function() {
             return [
-                'questionTags' => QuestionTagFactory::new(function() {
-                    return [
-                        'tag' => TagFactory::random(),
-                    ];
-                })->many(1, 5),
+                'tag' => TagFactory::random(),
+                'question' => QuestionFactory::random(),
             ];
         });
 
