diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 521ff7d4..faf9de16 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -7,6 +7,7 @@ use App\Entity\Question;
 use App\Entity\Tag;
 use App\Factory\AnswerFactory;
 use App\Factory\QuestionFactory;
+use App\Factory\QuestionTagFactory;
 use App\Factory\TagFactory;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Persistence\ObjectManager;
@@ -17,6 +18,10 @@ class AppFixtures extends Fixture
     {
         TagFactory::createMany(100);
 
+        QuestionTagFactory::createMany(10);
+
+        return;
+
         $questions = QuestionFactory::createMany(20, function() {
             return [
                 'tags' => TagFactory::randomRange(0, 5),
