diff --git a/src/Factory/QuestionTagFactory.php b/src/Factory/QuestionTagFactory.php
index 4fb5001..252de01 100644
--- a/src/Factory/QuestionTagFactory.php
+++ b/src/Factory/QuestionTagFactory.php
@@ -40,7 +40,7 @@ final class QuestionTagFactory extends ModelFactory
         return [
             'question' => QuestionFactory::new(),
             'tag' => TagFactory::new(),
-            'taggedAt' => self::faker()->datetime(),
+            'taggedAt' => \DateTimeImmutable::createFromMutable(self::faker()->datetime()),
         ];
     }
 
