diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 7988a663..eb60d0e1 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -20,7 +20,7 @@ class AppFixtures extends Fixture
 
         $questions = QuestionFactory::createMany(20, function() {
             return [
-                'questionTags' => QuestionTagFactory::new(),
+                'questionTags' => QuestionTagFactory::new()->many(1, 5),
             ];
         });
 
