diff --git a/src/Entity/QuestionTag.php b/src/Entity/QuestionTag.php
index 7a46e9e..235669d 100644
--- a/src/Entity/QuestionTag.php
+++ b/src/Entity/QuestionTag.php
@@ -34,6 +34,11 @@ class QuestionTag
      */
     private $taggedAt;
 
+    public function __construct()
+    {
+        $this->taggedAt = new \DateTimeImmutable();
+    }
+
     public function getId(): ?int
     {
         return $this->id;
