diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 32cc535..8d54821 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -20,9 +20,11 @@ class AppFixtures extends Fixture
 
         $questions = QuestionFactory::createMany(20, function() {
             return [
-                'questionTags' => QuestionTagFactory::new([
-                    'tag' => TagFactory::random()
-                ])->many(1, 5),
+                'questionTags' => QuestionTagFactory::new(function() {
+                    return [
+                        'tag' => TagFactory::random(),
+                    ];
+                })->many(1, 5),
             ];
         });
 
