diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index ca4963e..91c433a 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -42,6 +42,8 @@ class QuestionController extends AbstractController
             return $markdownParser->transformMarkdown($questionText);
         });
 
+        dd($markdownParser);
+
         return $this->render('question/show.html.twig', [
             'question' => ucwords(str_replace('-', ' ', $slug)),
             'questionText' => $parsedQuestionText,
