diff --git a/src/Controller/RegistrationController.php b/src/Controller/RegistrationController.php
index a90b5139..e4d60d6f 100644
--- a/src/Controller/RegistrationController.php
+++ b/src/Controller/RegistrationController.php
@@ -20,7 +20,7 @@ use SymfonyCasts\Bundle\VerifyEmail\VerifyEmailHelperInterface;
 class RegistrationController extends AbstractController
 {
     #[Route(path: '/register', name: 'app_register')]
-    public function register(Request $request, UserPasswordHasherInterface $userPasswordHasher, VerifyEmailHelperInterface $verifyEmailHelper, EntityManagerInterface $entityManager): Response
+    public function register(Request $request, UserPasswordHasherInterface $userPasswordHasher, VerifyEmailHelperInterface $verifyEmailHelper, EntityManagerInterface $entityManager, MailerInterface $mailer): Response
     {
         $user = new User();
         $form = $this->createForm(RegistrationFormType::class, $user);
@@ -44,11 +44,11 @@ class RegistrationController extends AbstractController
                 ['id' => $user->getId()]
             );
 
-            // TODO: in a real app, send this as an email!
             $signedUrl = $signatureComponents->getSignedUrl();
+            $this->sendVerificationEmail($mailer, $user, $signedUrl);
             $this->addFlash('success', sprintf(
-                'Confirm your email at: %s',
-                $signedUrl
+                'Confirm your email - the verify link was sent to %s',
+                $user->getEmail()
             ));
 
             return $this->redirectToRoute('app_homepage');
