diff --git a/src/Controller/RegistrationController.php b/src/Controller/RegistrationController.php
index 9eb6e19..00cd53f 100644
--- a/src/Controller/RegistrationController.php
+++ b/src/Controller/RegistrationController.php
@@ -19,7 +19,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class RegistrationController extends AbstractController
 {
     #[Route(path: '/register', name: 'app_register')]
-    public function register(Request $request, UserPasswordHasherInterface $userPasswordHasher, VerifyEmailHelperInterface $verifyEmailHelper, EntityManagerInterface $entityManager): Response
+    public function register(Request $request, UserPasswordHasherInterface $userPasswordHasher, VerifyEmailHelperInterface $verifyEmailHelper, EntityManagerInterface $entityManager, MailerInterface $mailer): Response
     {
         $user = new User();
         $form = $this->createForm(RegistrationFormType::class, $user);
@@ -44,11 +44,11 @@ class RegistrationController extends AbstractController
                 ['id' => $user->getId()]
             );
 
-            // TODO: in a real app, send this as an email!
             $signedUrl = $signatureComponents->getSignedUrl();
+            $this->sendVerificationEmail($mailer, $user, $signedUrl);
             $this->addFlash('success', sprintf(
-                'Confirm your email at: %s',
-                $signedUrl
+                'Confirm your email - the verify link was sent to %s',
+                $user->getEmail()
             ));
 
             return $this->redirectToRoute('app_homepage');
@@ -98,14 +98,14 @@ class RegistrationController extends AbstractController
     {
         $email = (new Email())
             ->from('hello@example.com')
-            ->to('you@example.com')
+            ->to($user->getEmail())
             //->cc('cc@example.com')
             //->bcc('bcc@example.com')
             //->replyTo('fabien@example.com')
             //->priority(Email::PRIORITY_HIGH)
-            ->subject('Time for Symfony Mailer!')
-            ->text('Sending emails is fun again!')
-            ->html('<p>See Twig integration for better HTML integration!</p>');
+            ->subject('Verify your email on Cauldron Overflow!')
+            ->text('Please, follow the link to verify your email!')
+            ->html(sprintf('<a href="%s">%s</a>', $signedUrl, $signedUrl));
 
         $mailer->send($email);
     }
