diff --git a/src/Controller/RegistrationController.php b/src/Controller/RegistrationController.php
index 5bd70e31..a90b5139 100644
--- a/src/Controller/RegistrationController.php
+++ b/src/Controller/RegistrationController.php
@@ -6,6 +6,9 @@ use App\Entity\User;
 use App\Form\RegistrationFormType;
 use App\Repository\UserRepository;
 use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
@@ -86,4 +89,20 @@ class RegistrationController extends AbstractController
     {
         return $this->render('registration/resend_verify_email.html.twig');
     }
+
+    private function sendVerificationEmail(MailerInterface $mailer, User $user, string $signedUrl)
+    {
+        $email = (new Email())
+            ->from('hello@example.com')
+            ->to($user->getEmail())
+            //->cc('cc@example.com')
+            //->bcc('bcc@example.com')
+            //->replyTo('fabien@example.com')
+            //->priority(Email::PRIORITY_HIGH)
+            ->subject('Verify your email on Cauldron Overflow!')
+            ->text('Please, follow the link to verify your email!')
+            ->html(sprintf('<a href="%s">%s</a>', $signedUrl, $signedUrl));
+
+        $mailer->send($email);
+    }
 }
