diff --git a/assets/css/app.css b/assets/css/app.css
index b7e59001..3ff383cc 100644
--- a/assets/css/app.css
+++ b/assets/css/app.css
@@ -1,3 +1,5 @@
+@import "~bootstrap";
+
 body {
     font-family: spartan;
     color: #444;
diff --git a/package.json b/package.json
index 968175f0..d1e1c367 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,7 @@
 {
     "devDependencies": {
         "@symfony/webpack-encore": "^0.28.2",
+        "bootstrap": "^4.4.1",
         "core-js": "^3.0.0",
         "jquery": "^3.4.1",
         "regenerator-runtime": "^0.13.2",
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 3157e2e2..d73bdb98 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -4,7 +4,6 @@
         <meta charset="UTF-8">
         <title>{% block title %}Welcome!{% endblock %}</title>
         {% block stylesheets %}
-            <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
             <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Spartan&display=swap">
             <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
             {{ encore_entry_link_tags('app') }}
diff --git a/yarn.lock b/yarn.lock
index d988c761..54027a60 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -1242,6 +1242,11 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
+bootstrap@^4.4.1:
+  version "4.4.1"
+  resolved "https://registry.yarnpkg.com/bootstrap/-/bootstrap-4.4.1.tgz#8582960eea0c5cd2bede84d8b0baf3789c3e8b01"
+  integrity sha512-tbx5cHubwE6e2ZG7nqM3g/FZ5PQEDMWmMGNrCUBVRPHXTJaH7CBDdsLeu3eCh3B1tzAxTnAbtmrzvWEvT2NNEA==
+
 brace-expansion@^1.1.7:
   version "1.1.11"
   resolved "https://registry.yarnpkg.com/brace-expansion/-/brace-expansion-1.1.11.tgz#3c7fcbf529d87226f3d2f52b966ff5271eb441dd"
