diff --git a/assets/js/app.js b/assets/js/app.js
index 1748c14..19dac89 100644
--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -11,4 +11,18 @@ import '../css/app.css';
 // Need jQuery? Install it with "yarn add jquery", then uncomment to import it.
 // import $ from 'jquery';
 
-console.log('Hello Webpack Encore! Edit me in assets/js/app.js');
+/**
+ * Simple (ugly) code to handle the comment vote up/down
+ */
+var $container = $('.js-vote-arrows');
+$container.find('a').on('click', function(e) {
+    e.preventDefault();
+    var $link = $(e.currentTarget);
+
+    $.ajax({
+        url: '/comments/10/vote/'+$link.data('direction'),
+        method: 'POST'
+    }).then(function(response) {
+        $container.find('.js-vote-total').text(response.votes);
+    });
+});
diff --git a/public/js/question_show.js b/public/js/question_show.js
deleted file mode 100644
index 7332aac..0000000
--- a/public/js/question_show.js
+++ /dev/null
@@ -1,15 +0,0 @@
-/**
- * Simple (ugly) code to handle the comment vote up/down
- */
-var $container = $('.js-vote-arrows');
-$container.find('a').on('click', function(e) {
-    e.preventDefault();
-    var $link = $(e.currentTarget);
-
-    $.ajax({
-        url: '/comments/10/vote/'+$link.data('direction'),
-        method: 'POST'
-    }).then(function(response) {
-        $container.find('.js-vote-total').text(response.votes);
-    });
-});
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index c7440ee..12e1f4d 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -56,9 +56,3 @@
     </ul>
 </div>
 {% endblock %}
-
-{% block javascripts %}
-    {{ parent() }}
-
-    <script src="{{ asset('js/question_show.js') }}"></script>
-{% endblock %}
