diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 1685462b..17d7db5b 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -44,6 +44,7 @@ class QuestionController extends AbstractController
         if ($this->isDebug) {
             $this->logger->info('We are in debug mode!');
         }
+        throw new \Exception('bad stuff happened!');
 
         $answers = [
             'Make sure your cat is sitting `purrrfectly` still 🤣',
