diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index f7dfcc06..1ebf1f05 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -6,7 +6,7 @@
 <div class="container">
     <div class="row">
         <div class="col-12">
-            <h2 class="my-4">Question</h2>
+            <h2 class="my-4">Question:</h2>
             <div style="box-shadow: 2px 3px 9px 4px rgba(0,0,0,0.04);">
                 <div class="q-container-show p-4">
                     <div class="row">
