diff --git a/templates/question/homepage.html.twig b/templates/question/homepage.html.twig
index 14273e9..b04b865 100644
--- a/templates/question/homepage.html.twig
+++ b/templates/question/homepage.html.twig
@@ -37,7 +37,7 @@
                 </div>
                 <a class="answer-link" href="{{ path('app_question_show', { slug: question.slug }) }}" style="color: #fff;">
                     <p class="q-display-response text-center p-3">
-                        <i class="fa fa-magic magic-wand"></i> 6 answers
+                        <i class="fa fa-magic magic-wand"></i> {{ question.answers|length}} answers
                     </p>
                 </a>
             </div>
