diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index 1714f86..c1f18c2 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -27,7 +27,8 @@ class QuestionRepository extends ServiceEntityRepository
     {
         return $this->addIsAskedQueryBuilder()
             ->orderBy('q.askedAt', 'DESC')
-            ->leftJoin('q.tags', 'tag')
+            ->leftJoin('q.questionTags', 'question_tag')
+            ->innerJoin('question_tag.tag', 'tag')
             ->addSelect('tag')
             ->getQuery()
             ->getResult()
