diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index ee79725..9968f20 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -16,6 +16,7 @@ class SecurityController extends AbstractController
     {
         return $this->render('security/login.html.twig', [
             'error' => $authenticationUtils->getLastAuthenticationError(),
+            'last_username' => $authenticationUtils->getLastUsername(),
         ]);
     }
 
diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index cb66135..bf03a19 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -15,7 +15,7 @@
 
                 <div class="col-12">
                     <label for="inputEmail">Email</label>
-                    <input type="email" name="email" id="inputEmail" class="form-control" required autofocus>
+                    <input type="email" name="email" id="inputEmail" class="form-control" value="{{ last_username }}" required autofocus>
                 </div>
                 <div class="col-12">
                     <label for="inputPassword">Password</label>
