diff --git a/src/Factory/CommentFactory.php b/src/Factory/CommentFactory.php
index 2de4ad67..ee960f65 100644
--- a/src/Factory/CommentFactory.php
+++ b/src/Factory/CommentFactory.php
@@ -40,7 +40,7 @@ final class CommentFactory extends ModelFactory
         return [
             'content' => self::faker()->text(),
             'username' => self::faker()->userName(),
-            'question' => QuestionFactory::new()->unpublished()->create(),
+            'question' => QuestionFactory::new()->unpublished(),
         ];
     }
 
