diff --git a/src/Factory/AnswerFactory.php b/src/Factory/AnswerFactory.php
index 4783e71d..548f9c04 100644
--- a/src/Factory/AnswerFactory.php
+++ b/src/Factory/AnswerFactory.php
@@ -38,11 +38,8 @@ final class AnswerFactory extends ModelFactory
     protected function getDefaults(): array
     {
         return [
-            // TODO add your default values here (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#model-factories)
             'content' => self::faker()->text(),
-            'username' => self::faker()->text(),
-            'createdAt' => null, // TODO add DATETIME ORM type manually
-            'updatedAt' => null, // TODO add DATETIME ORM type manually
+            'username' => self::faker()->userName(),
         ];
     }
 
