diff --git a/src/Factory/QuestionFactory.php b/src/Factory/QuestionFactory.php
index 1b9eee0..9745a7b 100644
--- a/src/Factory/QuestionFactory.php
+++ b/src/Factory/QuestionFactory.php
@@ -36,7 +36,7 @@ opening the front door and walking out! I've been out all afternoon
 Does anyone have a spell to call your pants back?
 EOF
             ,
-            'askedAt' => rand(1, 10) > 2 ? self::faker()->dateTimeBetween('-100 days', '-1 minute') : null,
+            'askedAt' => self::faker()->boolean(70) ? self::faker()->dateTimeBetween('-100 days', '-1 minute') : null,
             'votes' => rand(-20, 50),
         ];
     }
