diff --git a/src/Factory/QuestionFactory.php b/src/Factory/QuestionFactory.php
index 2450133..cf4573e 100644
--- a/src/Factory/QuestionFactory.php
+++ b/src/Factory/QuestionFactory.php
@@ -4,6 +4,7 @@ namespace App\Factory;
 
 use App\Entity\Question;
 use App\Repository\QuestionRepository;
+use Symfony\Component\String\Slugger\AsciiSlugger;
 use Zenstruck\Foundry\RepositoryProxy;
 use Zenstruck\Foundry\ModelFactory;
 use Zenstruck\Foundry\Proxy;
@@ -23,7 +24,6 @@ final class QuestionFactory extends ModelFactory
     {
         return [
             'name' => self::faker()->realText(50),
-            'slug' => self::faker()->slug,
             'question' => self::faker()->paragraphs(
                 self::faker()->numberBetween(1, 4),
                 true
@@ -37,7 +37,12 @@ final class QuestionFactory extends ModelFactory
     {
         // see https://github.com/zenstruck/foundry#initialization
         return $this
-            // ->beforeInstantiate(function(Question $question) {})
+            ->afterInstantiate(function(Question $question) {
+                if (!$question->getSlug()) {
+                    $slugger = new AsciiSlugger();
+                    $question->setSlug($slugger->slug($question->getName()));
+                }
+            })
         ;
     }
 
