diff --git a/src/Factory/AnswerFactory.php b/src/Factory/AnswerFactory.php
index ab82fed..b6258c3 100644
--- a/src/Factory/AnswerFactory.php
+++ b/src/Factory/AnswerFactory.php
@@ -42,7 +42,7 @@ final class AnswerFactory extends ModelFactory
             'username' => self::faker()->userName(),
             'createdAt' => self::faker()->dateTimeBetween('-1 year'),
             'votes' => rand(-20, 50),
-            'question' => QuestionFactory::new()->unpublished()->create(),
+            'question' => QuestionFactory::new()->unpublished(),
         ];
     }
 
