diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 9b87b7f9..69fb8613 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -13,6 +13,11 @@ class AppFixtures extends Fixture
     {
         QuestionFactory::new()->createMany(20);
 
+        QuestionFactory::new()
+            ->unpublished()
+            ->createMany(5)
+        ;
+
         $manager->flush();
     }
 }
diff --git a/src/Factory/QuestionFactory.php b/src/Factory/QuestionFactory.php
index cf4573ee..a4a0ca47 100644
--- a/src/Factory/QuestionFactory.php
+++ b/src/Factory/QuestionFactory.php
@@ -20,6 +20,11 @@ use Zenstruck\Foundry\Proxy;
  */
 final class QuestionFactory extends ModelFactory
 {
+    public function unpublished(): self
+    {
+        return $this->addState(['askedAt' => null]);
+    }
+
     protected function getDefaults(): array
     {
         return [
@@ -28,7 +33,7 @@ final class QuestionFactory extends ModelFactory
                 self::faker()->numberBetween(1, 4),
                 true
             ),
-            'askedAt' => self::faker()->boolean(70) ? self::faker()->dateTimeBetween('-100 days', '-1 minute') : null,
+            'askedAt' => self::faker()->dateTimeBetween('-100 days', '-1 minute'),
             'votes' => rand(-20, 50),
         ];
     }
