diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 162daf28..407d9b5e 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -21,9 +21,11 @@ class AppFixtures extends Fixture
             ->create()
         ;
 
-        AnswerFactory::createMany(100, [
-            'question' => QuestionFactory::random(),
-        ]);
+        AnswerFactory::createMany(100, function() {
+            return [
+                'question' => QuestionFactory::random(),
+            ];
+        });
 
         $manager->flush();
     }
