diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 407d9b5e..d6491a66 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -21,11 +21,7 @@ class AppFixtures extends Fixture
             ->create()
         ;
 
-        AnswerFactory::createMany(100, function() {
-            return [
-                'question' => QuestionFactory::random(),
-            ];
-        });
+        AnswerFactory::createMany(100);
 
         $manager->flush();
     }
diff --git a/src/Factory/AnswerFactory.php b/src/Factory/AnswerFactory.php
index 564aa01b..2d0aa9fe 100644
--- a/src/Factory/AnswerFactory.php
+++ b/src/Factory/AnswerFactory.php
@@ -42,6 +42,7 @@ final class AnswerFactory extends ModelFactory
             'username' => self::faker()->userName(),
             'createdAt' => self::faker()->dateTimeBetween('-1 year'),
             'votes' => rand(-20, 50),
+            'question' => QuestionFactory::random(),
         ];
     }
 
