diff --git a/src/Factory/QuestionFactory.php b/src/Factory/QuestionFactory.php
index e43b2da5..890f7a8c 100644
--- a/src/Factory/QuestionFactory.php
+++ b/src/Factory/QuestionFactory.php
@@ -36,7 +36,7 @@ opening the front door and walking out! I've been out all afternoon
 Does anyone have a spell to call your pants back?
 EOF
             ,
-            'askedAt' => rand(1, 10) > 2 ? new \DateTime(sprintf('-%d days', rand(1, 100))) : null,
+            'askedAt' => rand(1, 10) > 2 ? self::faker()->dateTimeBetween('-100 days', '-1 days') : null,
             'votes' => rand(-20, 50),
         ];
     }
