diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index d6491a66..162daf28 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -21,7 +21,9 @@ class AppFixtures extends Fixture
             ->create()
         ;
 
-        AnswerFactory::createMany(100);
+        AnswerFactory::createMany(100, [
+            'question' => QuestionFactory::random(),
+        ]);
 
         $manager->flush();
     }
