diff --git a/templates/base.html.twig b/templates/base.html.twig
index 71bdff5..718a6ea 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -13,7 +13,10 @@
         {% endblock %}
     </head>
     <body>
-        <nav class="navbar navbar-expand-lg navbar-light bg-light px-1">
+        <nav
+            class="navbar navbar-expand-lg navbar-light bg-light px-1"
+            {{ is_granted('ROLE_PREVIOUS_ADMIN') ? 'style="background-color: red !important"' }}
+        >
             <div class="container-fluid">
                 <a class="navbar-brand" href="{{ path('app_homepage') }}">
                     <p class="pl-2 d-inline font-weight-bold" style="color: #444;">
