diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 7080cb7a..ade1434b 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -34,9 +34,11 @@ class QuestionController extends AbstractController
             'Honestly, I like furry shoes better than MY cat',
             'Maybe... try saying the spell backwards?',
         ];
+        $questionText = 'I\'ve been turned into a cat, any thoughts on how to turn back? While I\'m adorable, I don\'t really care for cat food.';
 
         return $this->render('question/show.html.twig', [
             'question' => ucwords(str_replace('-', ' ', $slug)),
+            'questionText' => $questionText,
             'answers' => $answers,
         ]);
     }
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index dd54e7e8..855702d7 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -17,7 +17,7 @@
                             <h1 class="q-title-show">{{ question }}</h1>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
-                                <p class="d-inline">I've been turned into a cat, any thoughts on how to turn back? While I'm adorable, I don't really care for cat food.</p>
+                                <p class="d-inline">{{ questionText }}</p>
                                 <p class="pt-4"><strong>--Tisha</strong></p>
                             </div>
                         </div>
