diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 855702d7..4d32dfe0 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -17,7 +17,7 @@
                             <h1 class="q-title-show">{{ question }}</h1>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
-                                <p class="d-inline">{{ questionText }}</p>
+                                <p class="d-inline">{{ questionText|raw }}</p>
                                 <p class="pt-4"><strong>--Tisha</strong></p>
                             </div>
                         </div>
