diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index a45fe63..d564194 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -31,7 +31,7 @@ class QuestionController extends AbstractController
     public function show($slug, MarkdownParserInterface $markdownParser)
     {
         $answers = [
-            'Make sure your cat is sitting purrrfectly still 🤣',
+            'Make sure your cat is sitting `purrrfectly` still 🤣',
             'Honestly, I like furry shoes better than MY cat',
             'Maybe... try saying the spell backwards?',
         ];
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 4d32dfe..f7dfcc0 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -42,7 +42,7 @@
                         <img src="{{ asset('images/tisha.png') }}" width="50" height="50" alt="Tisha avatar">
                     </div>
                     <div class="mr-3 pt-2">
-                        {{ answer }}
+                        {{ answer|markdown }}
                         <p>-- Mallory</p>
                     </div>
                     <div class="vote-arrows flex-fill pt-2 js-vote-arrows" style="min-width: 90px;">
