diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index 86fec99..2e26758 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -157,7 +157,7 @@ class Question
     {
         if (!$this->answers->contains($answer)) {
             $this->answers[] = $answer;
-            //$answer->setQuestion($this);
+            $answer->setQuestion($this);
         }
 
         return $this;
