diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 107e601c..62d6615a 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -35,6 +35,9 @@ class AppFixtures extends Fixture
         $answer2->setContent('answer 1');
         $answer2->setUsername('weaverryan');
 
+        $question->addAnswer($answer1);
+        $question->addAnswer($answer2);
+
         $manager->persist($answer1);
         $manager->persist($answer2);
 
