diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 62d6615a..cdc9fbd6 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -27,20 +27,6 @@ class AppFixtures extends Fixture
             ];
         });
 
-        $question = QuestionFactory::createOne();
-        $answer1 = new Answer();
-        $answer1->setContent('answer 1');
-        $answer1->setUsername('weaverryan');
-        $answer2 = new Answer();
-        $answer2->setContent('answer 1');
-        $answer2->setUsername('weaverryan');
-
-        $question->addAnswer($answer1);
-        $question->addAnswer($answer2);
-
-        $manager->persist($answer1);
-        $manager->persist($answer2);
-
         $manager->flush();
     }
 }
