diff --git a/src/Entity/User.php b/src/Entity/User.php
index 1d55938a..8621a599 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -214,4 +214,12 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
 
         return $this;
     }
+
+    /**
+     * Fake method to return if the user has been verified.
+     */
+    public function isVerified(): bool
+    {
+        return false;
+    }
 }
