diff --git a/src/EventSubscriber/CheckVerifiedUserSubscriber.php b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
new file mode 100644
index 0000000..6b5e7c7
--- /dev/null
+++ b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class CheckVerifiedUserSubscriber implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents()
+    {
+    }
+}
