diff --git a/src/EventSubscriber/CheckVerifiedUserSubscriber.php b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
index 7ac66cb..6d22e33 100644
--- a/src/EventSubscriber/CheckVerifiedUserSubscriber.php
+++ b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
@@ -3,6 +3,7 @@
 namespace App\EventSubscriber;
 
 use App\Entity\User;
+use App\Security\AccountNotVerifiedAuthenticationException;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
@@ -25,9 +26,7 @@ class CheckVerifiedUserSubscriber implements EventSubscriberInterface
         }
 
         if (!$user->getIsVerified()) {
-            throw new CustomUserMessageAuthenticationException(
-                'Please verify your account before logging in.'
-            );
+            throw new AccountNotVerifiedAuthenticationException();
         }
     }
 
diff --git a/src/Security/AccountNotVerifiedAuthenticationException.php b/src/Security/AccountNotVerifiedAuthenticationException.php
new file mode 100644
index 0000000..1ab5e8a
--- /dev/null
+++ b/src/Security/AccountNotVerifiedAuthenticationException.php
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Security;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+class AccountNotVerifiedAuthenticationException extends AuthenticationException
+{
+}
