diff --git a/src/EventSubscriber/CheckVerifiedUserSubscriber.php b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
index 52d54fe8..3cfe267e 100644
--- a/src/EventSubscriber/CheckVerifiedUserSubscriber.php
+++ b/src/EventSubscriber/CheckVerifiedUserSubscriber.php
@@ -33,7 +33,9 @@ class CheckVerifiedUserSubscriber implements EventSubscriberInterface
 
     public function onLoginFailure(LoginFailureEvent $event)
     {
-        dd($event);
+        if (!$event->getException() instanceof AccountNotVerifiedAuthenticationException) {
+            return;
+        }
     }
 
     public static function getSubscribedEvents()
