diff --git a/templates/base.html.twig b/templates/base.html.twig
index 7cf4365b..d22fea84 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -31,6 +31,7 @@
                         </li>
                     </ul>
 
+                    <a class="nav-link text-black-50" href="{{ path('app_login') }}">Log In</a>
                     <a href="#" class="btn btn-dark">Sign up</a>
                 </div>
             </div>
