diff --git a/templates/base.html.twig b/templates/base.html.twig
index ce6f4a33..3f732721 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -31,6 +31,7 @@
                         </li>
                     </ul>
 
+                    <a class="nav-link text-black-50" href="{{ path('app_login') }}">Log In</a>
                     <button class="btn btn-dark">Sign up</button>
                 </div>
             </div>
