diff --git a/src/Twig/MarkdownExtension.php b/src/Twig/MarkdownExtension.php
index 028d07db..8aa7685d 100644
--- a/src/Twig/MarkdownExtension.php
+++ b/src/Twig/MarkdownExtension.php
@@ -22,7 +22,7 @@ class MarkdownExtension extends AbstractExtension
             // If your filter generates SAFE HTML, you should add a third
             // parameter: ['is_safe' => ['html']]
             // Reference: https://twig.symfony.com/doc/2.x/advanced.html#automatic-escaping
-            new TwigFilter('parse_markdown', [$this, 'parseMarkdown']),
+            new TwigFilter('parse_markdown', [$this, 'parseMarkdown'], ['is_safe' => ['html']]),
         ];
     }
 
