diff --git a/src/Command/RandomSpellCommand.php b/src/Command/RandomSpellCommand.php
index 7aaf13f0..402f6cd8 100644
--- a/src/Command/RandomSpellCommand.php
+++ b/src/Command/RandomSpellCommand.php
@@ -2,6 +2,7 @@
 
 namespace App\Command;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
@@ -12,6 +13,14 @@ use Symfony\Component\Console\Style\SymfonyStyle;
 class RandomSpellCommand extends Command
 {
     protected static $defaultName = 'app:random-spell';
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+
+        parent::__construct();
+    }
 
     protected function configure()
     {
@@ -47,6 +56,8 @@ class RandomSpellCommand extends Command
             $spell = strtoupper($spell);
         }
 
+        $this->logger->info('Casting spell: '.$spell);
+
         $io->success($spell);
 
         return 0;
