diff --git a/src/Twig/MarkdownExtension.php b/src/Twig/MarkdownExtension.php
index 746aa0e..028d07d 100644
--- a/src/Twig/MarkdownExtension.php
+++ b/src/Twig/MarkdownExtension.php
@@ -2,12 +2,20 @@
 
 namespace App\Twig;
 
+use App\Service\MarkdownHelper;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 use Twig\TwigFunction;
 
 class MarkdownExtension extends AbstractExtension
 {
+    private $markdownHelper;
+
+    public function __construct(MarkdownHelper $markdownHelper)
+    {
+        $this->markdownHelper = $markdownHelper;
+    }
+
     public function getFilters(): array
     {
         return [
@@ -20,6 +28,6 @@ class MarkdownExtension extends AbstractExtension
 
     public function parseMarkdown($value)
     {
-        return 'TEST';
+        return $this->markdownHelper->parse($value);
     }
 }
