diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 97e61c76..00f2e3a7 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -40,6 +40,9 @@ class AppFixtures extends Fixture
         $tag2 = new Tag();
         $tag2->setName('monster trucks');
 
+        $question->addTag($tag1);
+        $question->addTag($tag2);
+
         $manager->persist($tag1);
         $manager->persist($tag2);
 
