diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index cc695897..51cca240 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -33,7 +33,7 @@ class AppFixtures extends Fixture
             ];
         })->needsApproval()->many(20)->create();
 
-        $question = QuestionFactory::createOne();
+        $question = QuestionFactory::createOne()->object();
 
         $tag1 = new Tag();
         $tag1->setName('dinosaurs');
