diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 86fe7f00..85c5aa02 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -33,6 +33,7 @@ class AppFixtures extends Fixture
             ];
         })->needsApproval()->many(20)->create();
 
+        /** @var Question $question */
         $question = QuestionFactory::createOne()->object();
 
         $tag1 = new Tag();
@@ -40,6 +41,9 @@ class AppFixtures extends Fixture
         $tag2 = new Tag();
         $tag2->setName('monster trucks');
 
+        $question->addTag($tag1);
+        $question->addTag($tag2);
+
         $manager->persist($tag1);
         $manager->persist($tag2);
 
