diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index 1320b1a..07e5a96 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -19,10 +19,9 @@ class QuestionRepository extends ServiceEntityRepository
         parent::__construct($registry, Question::class);
     }
 
-    // /**
-    //  * @return Question[] Returns an array of Question objects
-    //  */
-    /*
+     /**
+      * @return Question[] Returns an array of Question objects
+      */
     public function findByExampleField($value)
     {
         return $this->createQueryBuilder('q')
@@ -34,7 +33,6 @@ class QuestionRepository extends ServiceEntityRepository
             ->getResult()
         ;
     }
-    */
 
     /*
     public function findOneBySomeField($value): ?Question
