diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 7cdec8ed..0db8e487 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -25,8 +25,12 @@ class QuestionController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage()
+    public function homepage(EntityManagerInterface $entityManager)
     {
+        $repository = $entityManager->getRepository(Question::class);
+        $questions = $repository->findAll();
+        dd($questions);
+
         return $this->render('question/homepage.html.twig');
     }
 
