diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index 07e5a969..177c73b1 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -22,7 +22,7 @@ class QuestionRepository extends ServiceEntityRepository
      /**
       * @return Question[] Returns an array of Question objects
       */
-    public function findByExampleField($value)
+    public function findAllAskedOrderedByNewest()
     {
         return $this->createQueryBuilder('q')
             ->andWhere('q.exampleField = :val')
