diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index f2b337f..f9a6ac1 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -73,19 +73,12 @@ EOF
     /**
      * @Route("/questions/{slug}", name="app_question_show")
      */
-    public function show($slug, EntityManagerInterface $entityManager)
+    public function show(Question $question, EntityManagerInterface $entityManager)
     {
         if ($this->isDebug) {
             $this->logger->info('We are in debug mode!');
         }
 
-        $repository = $entityManager->getRepository(Question::class);
-        /** @var Question|null $question */
-        $question = $repository->findOneBy(['slug' => $slug]);
-        if (!$question) {
-            throw $this->createNotFoundException(sprintf('no question found for slug "%s"', $slug));
-        }
-
         $answers = [
             'Make sure your cat is sitting `purrrfectly` still 🤣',
             'Honestly, I like furry shoes better than MY cat',
