diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index 5485e5c..ff204f6 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -4,6 +4,7 @@ namespace App\Repository;
 
 use App\Entity\Question;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\QueryBuilder;
 use Doctrine\Persistence\ManagerRegistry;
 
 /**
@@ -24,14 +25,20 @@ class QuestionRepository extends ServiceEntityRepository
       */
     public function findAllAskedOrderedByNewest()
     {
-        return $this->createQueryBuilder('q')
-            ->andWhere('q.askedAt IS NOT NULL')
+        $qb = $this->createQueryBuilder('q');
+
+        return $this->addIsAskedQueryBuilder($qb)
             ->orderBy('q.askedAt', 'DESC')
             ->getQuery()
             ->getResult()
         ;
     }
 
+    private function addIsAskedQueryBuilder(QueryBuilder $qb): QueryBuilder
+    {
+        return $qb->andWhere('q.askedAt IS NOT NULL');
+    }
+
     /*
     public function findOneBySomeField($value): ?Question
     {
