diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index ff204f6..6adb14a 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -25,18 +25,22 @@ class QuestionRepository extends ServiceEntityRepository
       */
     public function findAllAskedOrderedByNewest()
     {
-        $qb = $this->createQueryBuilder('q');
-
-        return $this->addIsAskedQueryBuilder($qb)
+        return $this->addIsAskedQueryBuilder()
             ->orderBy('q.askedAt', 'DESC')
             ->getQuery()
             ->getResult()
         ;
     }
 
-    private function addIsAskedQueryBuilder(QueryBuilder $qb): QueryBuilder
+    private function addIsAskedQueryBuilder(QueryBuilder $qb = null): QueryBuilder
+    {
+        return $this->getOrCreateQueryBuilder($qb)
+            ->andWhere('q.askedAt IS NOT NULL');
+    }
+
+    private function getOrCreateQueryBuilder(QueryBuilder $qb = null): QueryBuilder
     {
-        return $qb->andWhere('q.askedAt IS NOT NULL');
+        return $qb ?: $this->createQueryBuilder('q');
     }
 
     /*
