diff --git a/templates/question/homepage.html.twig b/templates/question/homepage.html.twig
index 121b967..8f16304 100644
--- a/templates/question/homepage.html.twig
+++ b/templates/question/homepage.html.twig
@@ -26,16 +26,16 @@
                             </div>
                         </div>
                         <div class="col">
-                            <a class="q-title" href="{{ path('app_question_show', { slug: 'reversing-a-spell' }) }}"><h2>Reversing a Spell</h2></a>
+                            <a class="q-title" href="{{ path('app_question_show', { slug: question.slug }) }}"><h2>{{ question.name }}</h2></a>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
-                                <p class="d-inline">I've been turned into a cat, any thoughts on how to turn back? While I'm adorable, I don't really care for cat food.</p>
+                                <p class="d-inline">{{ question.question|parse_markdown }}</p>
                                 <p class="pt-4"><strong>--Tisha</strong></p>
                             </div>
                         </div>
                     </div>
                 </div>
-                <a class="answer-link" href="{{ path('app_question_show', { slug: 'reversing-a-spell' }) }}" style="color: #fff;">
+                <a class="answer-link" href="{{ path('app_question_show', { slug: question.slug }) }}" style="color: #fff;">
                     <p class="q-display-response text-center p-3">
                         <i class="fa fa-magic magic-wand"></i> 6 answers
                     </p>
