diff --git a/src/Repository/QuestionRepository.php b/src/Repository/QuestionRepository.php
index 177c73b1..5485e5cd 100644
--- a/src/Repository/QuestionRepository.php
+++ b/src/Repository/QuestionRepository.php
@@ -25,10 +25,8 @@ class QuestionRepository extends ServiceEntityRepository
     public function findAllAskedOrderedByNewest()
     {
         return $this->createQueryBuilder('q')
-            ->andWhere('q.exampleField = :val')
-            ->setParameter('val', $value)
-            ->orderBy('q.id', 'ASC')
-            ->setMaxResults(10)
+            ->andWhere('q.askedAt IS NOT NULL')
+            ->orderBy('q.askedAt', 'DESC')
             ->getQuery()
             ->getResult()
         ;
