diff --git a/src/Repository/AnswerRepository.php b/src/Repository/AnswerRepository.php
index d9145365..93106c67 100644
--- a/src/Repository/AnswerRepository.php
+++ b/src/Repository/AnswerRepository.php
@@ -46,6 +46,7 @@ class AnswerRepository extends ServiceEntityRepository
        return $this->createQueryBuilder('answer')
            ->addCriteria(self::createApprovedCriteria())
            ->orderBy('answer.votes', 'DESC')
+           ->innerJoin('answer.question', 'question')
            ->setMaxResults(10)
            ->getQuery()
            ->getResult();
