diff --git a/src/Repository/AnswerRepository.php b/src/Repository/AnswerRepository.php
index 9571e5b8..61e1c26a 100644
--- a/src/Repository/AnswerRepository.php
+++ b/src/Repository/AnswerRepository.php
@@ -28,6 +28,7 @@ class AnswerRepository extends ServiceEntityRepository
             ->andWhere('answer.status = :approved')
             ->setParameter('approved', Answer::STATUS_APPROVED)
             ->orderBy('answer.votes', 'DESC')
+            ->innerJoin('answer.question', 'question')
             ->setMaxResults(10)
             ->getQuery()
             ->getResult()
