diff --git a/src/Repository/AnswerRepository.php b/src/Repository/AnswerRepository.php
index 93106c6..622e4c7 100644
--- a/src/Repository/AnswerRepository.php
+++ b/src/Repository/AnswerRepository.php
@@ -47,6 +47,7 @@ class AnswerRepository extends ServiceEntityRepository
            ->addCriteria(self::createApprovedCriteria())
            ->orderBy('answer.votes', 'DESC')
            ->innerJoin('answer.question', 'question')
+           ->addSelect('question')
            ->setMaxResults(10)
            ->getQuery()
            ->getResult();
