diff --git a/src/Repository/AnswerRepository.php b/src/Repository/AnswerRepository.php
index 622e4c76..d81e026e 100644
--- a/src/Repository/AnswerRepository.php
+++ b/src/Repository/AnswerRepository.php
@@ -48,6 +48,8 @@ class AnswerRepository extends ServiceEntityRepository
            ->orderBy('answer.votes', 'DESC')
            ->innerJoin('answer.question', 'question')
            ->addSelect('question')
+           ->andWhere('question.askedAt > :oneMonthAgo')
+           ->setParameter('oneMonthAgo', new \DateTimeImmutable('-1 month'))git
            ->setMaxResults(10)
            ->getQuery()
            ->getResult();
