diff --git a/src/Repository/AnswerRepository.php b/src/Repository/AnswerRepository.php
index 14c4b6a9..ad4552ae 100644
--- a/src/Repository/AnswerRepository.php
+++ b/src/Repository/AnswerRepository.php
@@ -50,7 +50,7 @@ class AnswerRepository extends ServiceEntityRepository
             ->addSelect('question');
 
         if ($search) {
-            $queryBuilder->andWhere('answer.content LIKE :searchTerm')
+            $queryBuilder->andWhere('answer.content LIKE :searchTerm OR question.question LIKE :searchTerm')
                 ->setParameter('searchTerm', '%'.$search.'%');
         }
 
