diff --git a/templates/answer/popularAnswers.html.twig b/templates/answer/popularAnswers.html.twig
index 296cceaa..f2bbd5b5 100644
--- a/templates/answer/popularAnswers.html.twig
+++ b/templates/answer/popularAnswers.html.twig
@@ -4,8 +4,18 @@
 
 {% block body %}
 <div class="container my-md-4">
-    <div>
+    <div class="d-flex justify-content-between">
         <h1>Most Popular Answers</h1>
+
+        <form action="{{ path('app_popular_answers') }}">
+            <input
+                type="search"
+                name="q"
+                class="form-control"
+                placeholder="Search..."
+                aria-label="Search"
+            >
+        </form>
     </div>
 
     <ul class="list-unstyled">
