diff --git a/config/routes.yaml b/config/routes.yaml
index c3283aa2..5939b360 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-#index:
-#    path: /
-#    controller: App\Controller\DefaultController::index
+index:
+    path: /
+    controller: App\Controller\QuestionController::homepage
diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
new file mode 100644
index 00000000..b2db7624
--- /dev/null
+++ b/src/Controller/QuestionController.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class QuestionController
+{
+    public function homepage()
+    {
+        return new Response('What a bewitching controller we have conjured!');
+    }
+}
