diff --git a/config/routes.yaml b/config/routes.yaml
index 5939b36..5033f30 100644
--- a/config/routes.yaml
+++ b/config/routes.yaml
@@ -1,3 +1,3 @@
-index:
-    path: /
-    controller: App\Controller\QuestionController::homepage
+#index:
+#    path: /
+#    controller: App\Controller\QuestionController::homepage
diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index b2db762..1a5131d 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -3,9 +3,13 @@
 namespace App\Controller;
 
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
 
 class QuestionController
 {
+    /**
+     * @Route("/")
+     */
     public function homepage()
     {
         return new Response('What a bewitching controller we have conjured!');
