diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 00a93f5e..edcd459d 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -16,10 +16,13 @@ class QuestionController
     }
 
     /**
-     * @Route("/questions/how-to-tie-my-shoes-with-magic")
+     * @Route("/questions/{slug}")
      */
-    public function show()
+    public function show($slug)
     {
-        return new Response('Future page to show a question!');
+        return new Response(sprintf(
+            'Future page to show the question "%s"!',
+            $slug
+        ));
     }
 }
