diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 17ce64f4..40df955d 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -30,12 +30,13 @@ class QuestionController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage(QuestionRepository $repository)
+    public function homepage(QuestionRepository $repository, Request $request)
     {
         $queryBuilder = $repository->createAskedOrderedByNewestQueryBuilder();
 
         $pagerfanta = new Pagerfanta(new QueryAdapter($queryBuilder));
         $pagerfanta->setMaxPerPage(5);
+        $pagerfanta->setCurrentPage($request->query->get('page', 1));
 
         return $this->render('question/homepage.html.twig', [
             'pager' => $pagerfanta,
