diff --git a/src/Factory/UserFactory.php b/src/Factory/UserFactory.php
index d3f1258..1ef00db 100644
--- a/src/Factory/UserFactory.php
+++ b/src/Factory/UserFactory.php
@@ -40,6 +40,7 @@ final class UserFactory extends ModelFactory
         return [
             'email' => self::faker()->email(),
             'firstName' => self::faker()->firstName(),
+            'plainPassword' => 'tada',
         ];
     }
 
