diff --git a/src/Entity/User.php b/src/Entity/User.php
index e5bf4925..187dce80 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -34,6 +34,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
      */
     private $firstName;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $password;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -126,4 +131,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface
 
         return $this;
     }
+
+    public function setPassword(string $password): self
+    {
+        $this->password = $password;
+
+        return $this;
+    }
 }
