diff --git a/src/Entity/User.php b/src/Entity/User.php
index 7d9d83ad..a6c087f4 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -33,10 +33,10 @@ class User implements UserInterface
     private array $roles = [];
 
     /**
-     * @var string The hashed password
+     * @var string|null The hashed password
      * @ORM\Column(type="string")
      */
-    private string $password;
+    private ?string $password = null;
 
     /**
      * @var string Non-mapped field
