diff --git a/src/Entity/Answer.php b/src/Entity/Answer.php
index 019288bd..f2f25fd1 100644
--- a/src/Entity/Answer.php
+++ b/src/Entity/Answer.php
@@ -96,6 +96,15 @@ class Answer
         return $this->question;
     }
 
+    public function getQuestionText(): string
+    {
+        if (!$this->getQuestion()) {
+            return '';
+        }
+
+        return (string) $this->getQuestion()->getQuestion();
+    }
+
     public function setQuestion(?Question $question): self
     {
         $this->question = $question;
diff --git a/templates/answer/_answer.html.twig b/templates/answer/_answer.html.twig
index eb745e49..c34ca421 100644
--- a/templates/answer/_answer.html.twig
+++ b/templates/answer/_answer.html.twig
@@ -7,7 +7,7 @@
             class="mb-1 link-secondary"
         >
             <strong>Question:</strong>
-            {{ answer.question.question }}
+            {{ answer.questionText }}
         </a>
     {% endif %}
     <div class="row">
