diff --git a/templates/answers/_answer.html.twig b/templates/answers/_answer.html.twig
index 1ae379b3..eb745e49 100644
--- a/templates/answers/_answer.html.twig
+++ b/templates/answers/_answer.html.twig
@@ -1,4 +1,15 @@
 <li class="mb-4">
+    {% if showQuestion|default(false) %}
+        <a
+            href="{{ path('app_question_show', {
+                slug: answer.question.slug
+            }) }}"
+            class="mb-1 link-secondary"
+        >
+            <strong>Question:</strong>
+            {{ answer.question.question }}
+        </a>
+    {% endif %}
     <div class="row">
         <div class="col-1">
             <img src="{{ asset('images/tisha.png') }}" width="50" height="50" alt="Tisha avatar">
diff --git a/templates/answers/popularAnswers.html.twig b/templates/answers/popularAnswers.html.twig
index 1745f514..c1101d98 100644
--- a/templates/answers/popularAnswers.html.twig
+++ b/templates/answers/popularAnswers.html.twig
@@ -9,7 +9,9 @@
 
         <ul class="list-unstyled">
             {% for answer in answers %}
-                {{ include('answers/_answer.html.twig') }}
+                {{ include('answers/_answer.html.twig', {
+                    showQuestion: true
+                }) }}
             {% endfor %}
         </ul>
     </div>
