diff --git a/src/Controller/AnswerController.php b/src/Controller/AnswerController.php
index 53a3c30..afb2c47 100644
--- a/src/Controller/AnswerController.php
+++ b/src/Controller/AnswerController.php
@@ -11,6 +11,14 @@ use Symfony\Component\Routing\Annotation\Route;
 
 class AnswerController extends AbstractController
 {
+    /**
+     * @Route("/answers/popular", name="app_popular_answers")
+     */
+    public function popularAnswers()
+    {
+        return $this->render('answer/popularAnswers.html.twig');
+    }
+
     /**
      * @Route("/answers/{id}/vote", methods="POST", name="answer_vote")
      */
diff --git a/templates/answer/popularAnswers.html.twig b/templates/answer/popularAnswers.html.twig
new file mode 100644
index 0000000..87b74ac
--- /dev/null
+++ b/templates/answer/popularAnswers.html.twig
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Popular Answers{% endblock %}
+
+{% block body %}
+<div class="container my-md-4">
+    <div class="row">
+        <h1>Most Popular Answers</h1>
+    </div>
+</div>
+{% endblock %}
diff --git a/templates/base.html.twig b/templates/base.html.twig
index d3b9d4c..cbaf723 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -20,6 +20,9 @@
             </a>
             <div class="collapse navbar-collapse">
                 <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ path('app_popular_answers') }}">Answers</a>
+                    </li>
                 </ul>
             </div>
 
