diff --git a/templates/question/homepage.html.twig b/templates/question/homepage.html.twig
index 2e1ed99..6913519 100644
--- a/templates/question/homepage.html.twig
+++ b/templates/question/homepage.html.twig
@@ -19,7 +19,7 @@
                 <div class="q-container p-4">
                     <div class="row">
                         <div class="col-2 text-center">
-                            <img src="{{ asset('images/tisha.png') }}" width="100" height="100">
+                            <img src="{{ asset('images/tisha.png') }}" width="100" height="100"  alt="Tisha avatar">
                             <div class="d-block mt-3 vote-arrows">
                                 <a class="vote-up" href="#"><i class="far fa-arrow-alt-circle-up"></i></a>
                                 <a class="vote-down" href="#"><i class="far fa-arrow-alt-circle-down"></i></a>
@@ -47,7 +47,7 @@
             <div class="q-container p-4">
                 <div class="row">
                     <div class="col-2 text-center">
-                        <img src="{{ asset('images/magic-photo.png') }}" width="100" height="100">
+                        <img src="{{ asset('images/magic-photo.png') }}" width="100" height="100"  alt="Jerry avatar">
                         <div class="d-block mt-3 vote-arrows">
                             <a class="vote-up" href="#"><i class="far fa-arrow-alt-circle-up"></i></a>
                             <a class="vote-down" href="#"><i class="far fa-arrow-alt-circle-down"></i></a>
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 12e1f4d..dd54e7e 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -11,7 +11,7 @@
                 <div class="q-container-show p-4">
                     <div class="row">
                         <div class="col-2 text-center">
-                            <img src="{{ asset('images/tisha.png') }}" width="100" height="100">
+                            <img src="{{ asset('images/tisha.png') }}" width="100" height="100"  alt="Tisha avatar">
                         </div>
                         <div class="col">
                             <h1 class="q-title-show">{{ question }}</h1>
@@ -39,7 +39,7 @@
             <li class="mb-4">
                 <div class="d-flex justify-content-center">
                     <div class="mr-2 pt-2">
-                        <img src="{{ asset('images/tisha.png') }}" width="50" height="50">
+                        <img src="{{ asset('images/tisha.png') }}" width="50" height="50" alt="Tisha avatar">
                     </div>
                     <div class="mr-3 pt-2">
                         {{ answer }}
