diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 76c24eb1..79dea800 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -80,20 +80,15 @@ EOF
         if (!$question) {
             throw $this->createNotFoundException(sprintf('no question found for slug "%s"', $slug));
         }
-        dd($question);
 
         $answers = [
             'Make sure your cat is sitting `purrrfectly` still 🤣',
             'Honestly, I like furry shoes better than MY cat',
             'Maybe... try saying the spell backwards?',
         ];
-        $questionText = 'I\'ve been turned into a cat, any *thoughts* on how to turn back? While I\'m **adorable**, I don\'t really care for cat food.';
-
-        $parsedQuestionText = $markdownHelper->parse($questionText);
 
         return $this->render('question/show.html.twig', [
-            'question' => ucwords(str_replace('-', ' ', $slug)),
-            'questionText' => $parsedQuestionText,
+            'question' => $question,
             'answers' => $answers,
         ]);
     }
diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index 6864965d..a8e56111 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -1,6 +1,6 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Question: {{ question }}{% endblock %}
+{% block title %}Question: {{ question.name }}{% endblock %}
 
 {% block body %}
 <div class="container">
@@ -26,10 +26,10 @@
                             </div>
                         </div>
                         <div class="col">
-                            <h1 class="q-title-show">{{ question }}</h1>
+                            <h1 class="q-title-show">{{ question.name }}</h1>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
-                                <p class="d-inline">{{ questionText|raw }}</p>
+                                <p class="d-inline">{{ question.question }}</p>
                                 <p class="pt-4"><strong>--Tisha</strong></p>
                             </div>
                         </div>
