diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index eaa1e8ef..76c24eb1 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -75,7 +75,11 @@ EOF
         }
 
         $repository = $entityManager->getRepository(Question::class);
+        /** @var Question|null $question */
         $question = $repository->findOneBy(['slug' => $slug]);
+        if (!$question) {
+            throw $this->createNotFoundException(sprintf('no question found for slug "%s"', $slug));
+        }
         dd($question);
 
         $answers = [
