diff --git a/templates/base.html.twig b/templates/base.html.twig
index 35696097..d0b14d7d 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -69,7 +69,7 @@
                         </div>
                     {% else %}
                         <a class="nav-link text-black-50" href="{{ path('app_login') }}">Log In</a>
-                        <a href="#" class="btn btn-dark">Sign up</a>
+                        <a href="{{ path('app_register') }}" class="btn btn-dark">Sign up</a>
                     {% endif %}
                 </div>
             </div>
