diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index cb66135..ab38a42 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -22,6 +22,12 @@
                     <input type="password" name="password" id="inputPassword" class="form-control" required>
                 </div>
 
+                <div class="form-check mb-3">
+                    <label>
+                        <input type="checkbox" name="_remember_me" class="form-check-input"> Remember me
+                    </label>
+                </div>
+
                 <input type="hidden" name="_csrf_token"
                     value="{{ csrf_token('authenticate') }}"
                 >
