diff --git a/config/packages/security.yaml b/config/packages/security.yaml
index c78e40d..f53f688 100644
--- a/config/packages/security.yaml
+++ b/config/packages/security.yaml
@@ -27,6 +27,7 @@ security:
 
             remember_me:
                 secret: '%kernel.secret%'
+                always_remember_me: true
 
             # activate different ways to authenticate
             # https://symfony.com/doc/current/security.html#firewalls-authentication
diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index ab38a42..cb66135 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -22,12 +22,6 @@
                     <input type="password" name="password" id="inputPassword" class="form-control" required>
                 </div>
 
-                <div class="form-check mb-3">
-                    <label>
-                        <input type="checkbox" name="_remember_me" class="form-check-input"> Remember me
-                    </label>
-                </div>
-
                 <input type="hidden" name="_csrf_token"
                     value="{{ csrf_token('authenticate') }}"
                 >
