diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 4cb34417..6ab8f153 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -9,7 +9,7 @@ use Symfony\Component\Routing\Annotation\Route;
 class QuestionController extends AbstractController
 {
     /**
-     * @Route("/")
+     * @Route("/", name="app_homepage")
      */
     public function homepage()
     {
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 2b32bf7e..4f1651e8 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -12,7 +12,7 @@
     </head>
     <body>
         <nav class="navbar navbar-light bg-light" style="height: 100px;">
-          <a class="navbar-brand" href="#">
+          <a class="navbar-brand" href="{{ path('app_homepage') }}">
             <i style="color: #444; font-size: 2rem;" class="pb-1 fad fa-cauldron"></i>
             <p class="pl-2 d-inline font-weight-bold" style="color: #444;">Cauldron Overflow</p>
           </a>
