diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 1c6c66cc..b03d9efd 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Entity\Question;
 use App\Service\MarkdownHelper;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
@@ -33,6 +34,28 @@ class QuestionController extends AbstractController
      */
     public function new()
     {
+        $question = new Question();
+        $question->setName('Missing pants')
+            ->setSlug('missing-pants-'.rand(0, 1000))
+            ->setQuestion(<<<EOF
+Hi! So... I'm having a *weird* day. Yesterday, I cast a spell
+to make my dishes wash themselves. But while I was casting it,
+I slipped a little and I think `I also hit my pants with the spell`.
+
+When I woke up this morning, I caught a quick glimpse of my pants
+opening the front door and walking out! I've been out all afternoon
+(with no pants mind you) searching for them.
+
+Does anyone have a spell to call your pants back?
+EOF
+            );
+
+        if (rand(1, 10) > 2) {
+            $question->setAskedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
+        }
+
+        dd($question);
+
         return new Response('Time for some Doctrine magic!');
     }
 
