diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 8f9d433c..8678ff67 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -52,4 +52,12 @@ class QuestionController extends AbstractController
             'answers' => $answers,
         ]);
     }
+
+    /**
+     * @Route("/questions/new")
+     */
+    public function new()
+    {
+        return new Response('Time for some Doctrine magic!');
+    }
 }
