diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 092de65..1ee92c6 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -4,6 +4,7 @@ namespace App\Controller;
 
 use App\Entity\Question;
 use App\Service\MarkdownHelper;
+use Doctrine\ORM\EntityManagerInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
@@ -32,7 +33,7 @@ class QuestionController extends AbstractController
     /**
      * @Route("/questions/new")
      */
-    public function new()
+    public function new(EntityManagerInterface $entityManager)
     {
         $question = new Question();
         $question->setName('Missing pants')
@@ -54,7 +55,14 @@ EOF
             $question->setAskedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
         }
 
-        return new Response('Time for some Doctrine magic!');
+        $entityManager->persist($question);
+        $entityManager->flush();
+
+        return new Response(sprintf(
+            'Well hallo! The shiny new question is id #%d, slug: %s',
+            $question->getId(),
+            $question->getSlug()
+        ));
     }
 
     /**
