diff --git a/config/packages/scheb_2fa.yaml b/config/packages/scheb_2fa.yaml
index 933774d6..f9ad742d 100644
--- a/config/packages/scheb_2fa.yaml
+++ b/config/packages/scheb_2fa.yaml
@@ -9,3 +9,4 @@ scheb_two_factor:
     totp:
         enabled: true
         issuer: 'Cauldron Overflow'
+        template: security/2fa_form.html.twig
diff --git a/templates/security/2fa_form.html.twig b/templates/security/2fa_form.html.twig
new file mode 100644
index 00000000..fbd408a3
--- /dev/null
+++ b/templates/security/2fa_form.html.twig
@@ -0,0 +1,19 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Two Factor Auth{% endblock %}
+
+{% block body %}
+<div class="container">
+    <div class="row">
+        <div class="login-form bg-light mt-4 p-4">
+            <h1 class="h3 mb-3 font-weight-normal">Two Factor Authentication</h1>
+
+            <p>
+                Open your Authenticator app and type in the number.
+            </p>
+
+            FORM TODO
+        </div>
+    </div>
+</div>
+{% endblock %}
