diff --git a/src/Entity/User.php b/src/Entity/User.php
index 0e2c348f..ac4f2312 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -254,11 +254,6 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
         // TODO: Implement getTotpAuthenticationConfiguration() method.
     }
 
-    public function getTotpSecret(): ?string
-    {
-        return $this->totpSecret;
-    }
-
     public function setTotpSecret(?string $totpSecret): self
     {
         $this->totpSecret = $totpSecret;
