diff --git a/src/Entity/User.php b/src/Entity/User.php
index 7e97fff6..0e2c348f 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -61,6 +61,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
      */
     private $isVerified = false;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $totpSecret;
+
     public function __construct()
     {
         $this->questions = new ArrayCollection();
@@ -248,4 +253,16 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
     {
         // TODO: Implement getTotpAuthenticationConfiguration() method.
     }
+
+    public function getTotpSecret(): ?string
+    {
+        return $this->totpSecret;
+    }
+
+    public function setTotpSecret(?string $totpSecret): self
+    {
+        $this->totpSecret = $totpSecret;
+
+        return $this;
+    }
 }
