diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index 4ce2b551..ffadb4a1 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -44,6 +44,6 @@ class SecurityController extends BaseController
             $entityManager->flush();
         }
 
-        dd($user);
+        dd($totpAuthenticator->getQRContent($user));
     }
 }
