diff --git a/src/Entity/User.php b/src/Entity/User.php
index 682cc45b..716bf1b2 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -56,6 +56,11 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
      */
     private $questions;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $totpSecret;
+
     public function __construct()
     {
         $this->questions = new ArrayCollection();
@@ -239,4 +244,16 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
     {
         // TODO: Implement getTotpAuthenticationConfiguration() method.
     }
+
+    public function getTotpSecret(): ?string
+    {
+        return $this->totpSecret;
+    }
+
+    public function setTotpSecret(?string $totpSecret): self
+    {
+        $this->totpSecret = $totpSecret;
+
+        return $this;
+    }
 }
