diff --git a/src/Entity/User.php b/src/Entity/User.php
index 3ffe2ce..faffab0 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -257,11 +257,6 @@ class User implements UserInterface, PasswordAuthenticatedUserInterface, TwoFact
         // TODO: Implement getTotpAuthenticationConfiguration() method.
     }
 
-    public function getTotpSecret(): ?string
-    {
-        return $this->totpSecret;
-    }
-
     public function setTotpSecret(?string $totpSecret): self
     {
         $this->totpSecret = $totpSecret;
