diff --git a/src/Controller/SecurityController.php b/src/Controller/SecurityController.php
index b514fc3f..95f85aff 100644
--- a/src/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -4,6 +4,7 @@ namespace App\Controller;
 
 use Doctrine\ORM\EntityManagerInterface;
 use Scheb\TwoFactorBundle\Security\TwoFactor\Provider\Totp\TotpAuthenticatorInterface;
+use Scheb\TwoFactorBundle\Security\TwoFactor\QrCode\QrCodeGenerator;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
@@ -54,4 +55,16 @@ class SecurityController extends BaseController
 
         dd($totpAuthenticator->getQRContent($user));
     }
+
+    /**
+     * @Route("/authentication/2fa/qr-code", name="app_qr_code")
+     */
+    public function displayGoogleAuthenticatorQrCode(QrCodeGenerator $qrCodeGenerator)
+    {
+        // $qrCode is provided by the endroid/qr-code library. See the docs how to customize the look of the QR code:
+        // https://github.com/endroid/qr-code
+        $qrCode = $qrCodeGenerator->getTotpQrCode($this->getUser());
+
+        return new Response($qrCode->writeString(), 200, ['Content-Type' => 'image/png']);
+    }
 }
