diff --git a/src/Controller/CommentController.php b/src/Controller/CommentController.php
index 98af1c4b..3f12122d 100644
--- a/src/Controller/CommentController.php
+++ b/src/Controller/CommentController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\Routing\Annotation\Route;
@@ -11,14 +12,16 @@ class CommentController extends AbstractController
     /**
      * @Route("/comments/{id}/vote/{direction<up|down>}", methods="POST")
      */
-    public function commentVote($id, $direction)
+    public function commentVote($id, $direction, LoggerInterface $logger)
     {
         // todo - use id to query the database
 
         // use real logic here to save this to the database
         if ($direction === 'up') {
+            $logger->info('Voting up!');
             $currentVoteCount = rand(7, 100);
         } else {
+            $logger->info('Voting down!');
             $currentVoteCount = rand(0, 5);
         }
 
