diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index f5edcfa0..3caeb8fb 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -5,15 +5,21 @@ namespace App\Controller;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Twig\Environment;
 
 class QuestionController extends AbstractController
 {
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage()
+    public function homepage(Environment $twigEnvironment)
     {
-        return $this->render('question/homepage.html.twig');
+        // fun example of using the Twig service directly!
+        $html = $twigEnvironment->render('question/homepage.html.twig');
+
+        return new Response($html);
+
+        //return $this->render('question/homepage.html.twig');
     }
 
     /**
