diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 0e3d2d86..a9dfbb44 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -3,11 +3,9 @@
 namespace App\Controller;
 
 use App\Service\MarkdownHelper;
-use Knp\Bundle\MarkdownBundle\MarkdownParserInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
-use Symfony\Contracts\Cache\CacheInterface;
 use Twig\Environment;
 
 class QuestionController extends AbstractController
@@ -30,7 +28,7 @@ class QuestionController extends AbstractController
     /**
      * @Route("/questions/{slug}", name="app_question_show")
      */
-    public function show($slug, MarkdownParserInterface $markdownParser, CacheInterface $cache, MarkdownHelper $markdownHelper)
+    public function show($slug, MarkdownHelper $markdownHelper)
     {
         $answers = [
             'Make sure your cat is sitting `purrrfectly` still 🤣',
