diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 8459573..5fddfd0 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -11,7 +11,7 @@ class MarkdownHelper
     private $cache;
     private $isDebug;
 
-    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache, $isDebug)
+    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache, bool $isDebug)
     {
         $this->markdownParser = $markdownParser;
         $this->cache = $cache;
