diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 8b567e5..e586f7e 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -9,15 +9,21 @@ class MarkdownHelper
 {
     private $markdownParser;
     private $cache;
+    private $isDebug;
 
-    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache)
+    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache, bool $isDebug)
     {
         $this->markdownParser = $markdownParser;
         $this->cache = $cache;
+        $this->isDebug = $isDebug;
     }
 
     public function parse(string $source): string
     {
+        if ($this->isDebug) {
+            return $this->markdownParser->transformMarkdown($source);
+        }
+
         return $this->cache->get('markdown_'.md5($source), function() use ($source) {
             return $this->markdownParser->transformMarkdown($source);
         });
