diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index e586f7e..5fddfd0 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -16,6 +16,7 @@ class MarkdownHelper
         $this->markdownParser = $markdownParser;
         $this->cache = $cache;
         $this->isDebug = $isDebug;
+        dump($isDebug);
     }
 
     public function parse(string $source): string
