diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 206e57b6..285aef79 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -21,11 +21,11 @@ class AppFixtures extends Fixture
         ;
 
         $question = QuestionFactory::createOne()->object();
-        dd($question);
 
         $answer = new Answer();
         $answer->setContent('This question is the best? I wish... I knew the answer.');
         $answer->setUsername('weaverryan');
+        $answer->setQuestion($question);
 
         $manager->persist($answer);
 
