diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 3f434a0f..45477690 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -21,11 +21,11 @@ class AppFixtures extends Fixture
         ;
 
         $question = QuestionFactory::createOne()->object();
-        dd($question);
 
         $comment = new Comment();
         $comment->setContent('This question is the best? I wish... I knew the answer.');
         $comment->setUsername('weaverryan');
+        $comment->setQuestion($question);
 
         $manager->persist($comment);
 
