diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index a9574770..575ef012 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -28,6 +28,8 @@ class AppFixtures extends Fixture
         $question->setName('How to un-disappear your wallet.');
         $question->setQuestion('... I should not have done this...');
 
+        $answer->setQuestion($question);
+
         $manager->persist($answer);
         $manager->persist($question);
 
