diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 0ade76a2..a9574770 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -2,6 +2,7 @@
 
 namespace App\DataFixtures;
 
+use App\Entity\Answer;
 use App\Entity\Question;
 use App\Factory\QuestionFactory;
 use Doctrine\Bundle\FixturesBundle\Fixture;
@@ -19,6 +20,17 @@ class AppFixtures extends Fixture
             ->create()
         ;
 
+        $answer = new Answer();
+        $answer->setContent('This question is the best? I wish... I knew the answer.');
+        $answer->setUsername('weaverryan');
+
+        $question = new Question();
+        $question->setName('How to un-disappear your wallet.');
+        $question->setQuestion('... I should not have done this...');
+
+        $manager->persist($answer);
+        $manager->persist($question);
+
         $manager->flush();
     }
 }
