diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index cdc9fbd6..44d028c8 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -26,6 +26,11 @@ class AppFixtures extends Fixture
                 'question' => $questions[array_rand($questions)]
             ];
         });
+        AnswerFactory::new(function() use ($questions) {
+            return [
+                'question' => $questions[array_rand($questions)]
+            ];
+        })->needsApproval()->many(20)->create();
 
         $manager->flush();
     }
diff --git a/src/Factory/AnswerFactory.php b/src/Factory/AnswerFactory.php
index e51d1aa9..fcbc98fa 100644
--- a/src/Factory/AnswerFactory.php
+++ b/src/Factory/AnswerFactory.php
@@ -35,6 +35,11 @@ final class AnswerFactory extends ModelFactory
         // TODO inject services if required (https://symfony.com/bundles/ZenstruckFoundryBundle/current/index.html#factories-as-services)
     }
 
+    public function needsApproval(): self
+    {
+        return $this->addState(['status' => Answer::STATUS_NEEDS_APPROVAL]);
+    }
+
     protected function getDefaults(): array
     {
         return [
