diff --git a/migrations/Version20210902182514.php b/migrations/Version20210902182514.php
new file mode 100644
index 0000000..c8c8d08
--- /dev/null
+++ b/migrations/Version20210902182514.php
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210902182514 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE answer ADD status VARCHAR(15) NOT NULL');
+        $this->addSql('ALTER TABLE answer RENAME INDEX idx_9474526c1e27f6bf TO IDX_DADD4A251E27F6BF');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE answer DROP status');
+        $this->addSql('ALTER TABLE answer RENAME INDEX idx_dadd4a251e27f6bf TO IDX_9474526C1E27F6BF');
+    }
+}
diff --git a/src/Entity/Answer.php b/src/Entity/Answer.php
index f5e2751..2c80fce 100644
--- a/src/Entity/Answer.php
+++ b/src/Entity/Answer.php
@@ -41,6 +41,11 @@ class Answer
      */
     private $question;
 
+    /**
+     * @ORM\Column(type="string", length=15)
+     */
+    private $status;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -93,4 +98,16 @@ class Answer
 
         return $this;
     }
+
+    public function getStatus(): ?string
+    {
+        return $this->status;
+    }
+
+    public function setStatus(string $status): self
+    {
+        $this->status = $status;
+
+        return $this;
+    }
 }
