diff --git a/src/Entity/Answer.php b/src/Entity/Answer.php
index 2c80fce3..21aaa773 100644
--- a/src/Entity/Answer.php
+++ b/src/Entity/Answer.php
@@ -11,6 +11,10 @@ use Gedmo\Timestampable\Traits\TimestampableEntity;
  */
 class Answer
 {
+    public const STATUS_NEEDS_APPROVAL = 'needs_approval';
+    public const STATUS_SPAM = 'spam';
+    public const STATUS_APPROVED = 'approved';
+
     use TimestampableEntity;
 
     /**
@@ -44,7 +48,7 @@ class Answer
     /**
      * @ORM\Column(type="string", length=15)
      */
-    private $status;
+    private $status = self::STATUS_NEEDS_APPROVAL;
 
     public function getId(): ?int
     {
@@ -106,6 +110,10 @@ class Answer
 
     public function setStatus(string $status): self
     {
+        if (!in_array($status, [self::STATUS_NEEDS_APPROVAL, self::STATUS_SPAM, self::STATUS_APPROVED])) {
+            throw new \InvalidArgumentException(sprintf('Invalid status "%s"', $status));
+        }
+
         $this->status = $status;
 
         return $this;
diff --git a/src/Factory/AnswerFactory.php b/src/Factory/AnswerFactory.php
index 2d0aa9fe..f87445f9 100644
--- a/src/Factory/AnswerFactory.php
+++ b/src/Factory/AnswerFactory.php
@@ -43,6 +43,7 @@ final class AnswerFactory extends ModelFactory
             'createdAt' => self::faker()->dateTimeBetween('-1 year'),
             'votes' => rand(-20, 50),
             'question' => QuestionFactory::random(),
+            'status' => Answer::STATUS_APPROVED,
         ];
     }
 
