diff --git a/rector.php b/rector.php
index 8d74e396..a2b32009 100644
--- a/rector.php
+++ b/rector.php
@@ -21,14 +21,7 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     );
     // endregion
 
-    // Define what rule sets will be applied
-    $containerConfigurator->import(DoctrineSetList::ANNOTATIONS_TO_ATTRIBUTES);
-    $containerConfigurator->import(SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES);
-    $containerConfigurator->import(SensiolabsSetList::FRAMEWORK_EXTRA_61);
-
-    // get services (needed for register a single rule)
-    // $services = $containerConfigurator->services();
-
-    // register a single rule
-    // $services->set(TypedPropertyRector::class);
+    $containerConfigurator->import(SymfonyLevelSetList::UP_TO_SYMFONY_60);
+    $containerConfigurator->import(SymfonySetList::SYMFONY_CODE_QUALITY);
+    $containerConfigurator->import(SymfonySetList::SYMFONY_CONSTRUCTOR_INJECTION);
 };
