diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 51cca24..91714fd 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -7,6 +7,7 @@ use App\Entity\Question;
 use App\Entity\Tag;
 use App\Factory\AnswerFactory;
 use App\Factory\QuestionFactory;
+use App\Factory\TagFactory;
 use Doctrine\Bundle\FixturesBundle\Fixture;
 use Doctrine\Persistence\ObjectManager;
 
@@ -14,7 +15,11 @@ class AppFixtures extends Fixture
 {
     public function load(ObjectManager $manager)
     {
-        $questions = QuestionFactory::createMany(20);
+        TagFactory::createMany(100);
+
+        $questions = QuestionFactory::createMany(20, [
+            'tags' => TagFactory::randomRange(0, 5),
+        ]);
 
         QuestionFactory::new()
             ->unpublished()
@@ -33,19 +38,6 @@ class AppFixtures extends Fixture
             ];
         })->needsApproval()->many(20)->create();
 
-        $question = QuestionFactory::createOne()->object();
-
-        $tag1 = new Tag();
-        $tag1->setName('dinosaurs');
-        $tag2 = new Tag();
-        $tag2->setName('monster trucks');
-
-        $tag1->addQuestion($question);
-        $tag2->addQuestion($question);
-
-        $manager->persist($tag1);
-        $manager->persist($tag2);
-
         $manager->flush();
     }
 }
