diff --git a/src/DataFixtures/AppFixtures.php b/src/DataFixtures/AppFixtures.php
index 91714fdd..d6aa7292 100644
--- a/src/DataFixtures/AppFixtures.php
+++ b/src/DataFixtures/AppFixtures.php
@@ -17,9 +17,11 @@ class AppFixtures extends Fixture
     {
         TagFactory::createMany(100);
 
-        $questions = QuestionFactory::createMany(20, [
-            'tags' => TagFactory::randomRange(0, 5),
-        ]);
+        $questions = QuestionFactory::createMany(20, function() {
+            return [
+                'tags' => TagFactory::randomRange(0, 5),
+            ];
+        });
 
         QuestionFactory::new()
             ->unpublished()
