diff --git a/migrations/Version20200709153558.php b/migrations/Version20200709153558.php
index 38a9e97..7d310f6 100644
--- a/migrations/Version20200709153558.php
+++ b/migrations/Version20200709153558.php
@@ -20,7 +20,8 @@ final class Version20200709153558 extends AbstractMigration
     public function up(Schema $schema) : void
     {
         // this up() migration is auto-generated, please modify it to your needs
-        $this->addSql('ALTER TABLE question ADD created_at DATETIME NOT NULL, ADD updated_at DATETIME NOT NULL');
+        $this->addSql('ALTER TABLE question ADD created_at DATETIME DEFAULT NULL, ADD updated_at DATETIME DEFAULT NULL');
+        $this->addSql('UPDATE question SET created_at = NOW(), updated_at = NOW()');
     }
 
     public function down(Schema $schema) : void
