diff --git a/src/Service/MarkdownHelper.php b/src/Service/MarkdownHelper.php
index 5b2d991a..0e172e79 100644
--- a/src/Service/MarkdownHelper.php
+++ b/src/Service/MarkdownHelper.php
@@ -4,6 +4,7 @@ namespace App\Service;
 
 use Knp\Bundle\MarkdownBundle\MarkdownParserInterface;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Security\Core\Security;
 use Symfony\Contracts\Cache\CacheInterface;
 
 class MarkdownHelper
@@ -12,13 +13,15 @@ class MarkdownHelper
     private $cache;
     private $isDebug;
     private $logger;
+    private Security $security;
 
-    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache, bool $isDebug, LoggerInterface $mdLogger)
+    public function __construct(MarkdownParserInterface $markdownParser, CacheInterface $cache, bool $isDebug, LoggerInterface $mdLogger, Security $security)
     {
         $this->markdownParser = $markdownParser;
         $this->cache = $cache;
         $this->isDebug = $isDebug;
         $this->logger = $mdLogger;
+        $this->security = $security;
     }
 
     public function parse(string $source): string
@@ -27,6 +30,12 @@ class MarkdownHelper
             $this->logger->info('Meow!');
         }
 
+        if ($this->security->getUser()) {
+            $this->logger->info('Rendering markdown for {user}', [
+                'user' => $this->security->getUser()->getUserIdentifier()
+            ]);
+        }
+
         if ($this->isDebug) {
             return $this->markdownParser->transformMarkdown($source);
         }
