diff --git a/src/Controller/AnswerController.php b/src/Controller/AnswerController.php
index f7932e39..26c4eb75 100644
--- a/src/Controller/AnswerController.php
+++ b/src/Controller/AnswerController.php
@@ -6,6 +6,7 @@ use App\Entity\Answer;
 use App\Repository\AnswerRepository;
 use Doctrine\ORM\EntityManagerInterface;
 use Psr\Log\LoggerInterface;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
@@ -28,6 +29,7 @@ class AnswerController extends AbstractController
 
     /**
      * @Route("/answers/{id}/vote", methods="POST", name="answer_vote")
+     * @IsGranted("IS_AUTHENTICATED_REMEMBERED")
      */
     public function answerVote(Answer $answer, LoggerInterface $logger, Request $request, EntityManagerInterface $entityManager)
     {
diff --git a/templates/answer/_answer.html.twig b/templates/answer/_answer.html.twig
index 64567f1d..75c9a670 100644
--- a/templates/answer/_answer.html.twig
+++ b/templates/answer/_answer.html.twig
@@ -20,6 +20,7 @@
         </div>
         <div class="col-2 text-end">
             <small>{{ answer.createdAt|ago }}</small>
+            {% if is_granted('IS_AUTHENTICATED_REMEMBERED') %}
             <div
                 class="vote-arrows"
                 {{ stimulus_controller('answer-vote', {
@@ -42,6 +43,7 @@
                 ><i class="far fa-arrow-alt-circle-down"></i></button>
                 <span><span {{ stimulus_target('answer-vote', 'voteTotal') }}>{{ answer.votes }}</span></span>
             </div>
+            {% endif %}
         </div>
     </div>
 </li>
