diff --git a/src/Entity/User.php b/src/Entity/User.php
index 0da9d7d..a095f14 100644
--- a/src/Entity/User.php
+++ b/src/Entity/User.php
@@ -29,6 +29,11 @@ class User implements UserInterface
      */
     private $roles = [];
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $firstName;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -111,4 +116,16 @@ class User implements UserInterface
         // If you store any temporary, sensitive data on the user, clear it here
         // $this->plainPassword = null;
     }
+
+    public function getFirstName(): ?string
+    {
+        return $this->firstName;
+    }
+
+    public function setFirstName(string $firstName): self
+    {
+        $this->firstName = $firstName;
+
+        return $this;
+    }
 }
