diff --git a/src/Controller/AnswerController.php b/src/Controller/AnswerController.php
index 846b8f83..cc8d3f56 100644
--- a/src/Controller/AnswerController.php
+++ b/src/Controller/AnswerController.php
@@ -34,7 +34,7 @@ class AnswerController extends AbstractController
     public function answerVote(Answer $answer, LoggerInterface $logger, Request $request, EntityManagerInterface $entityManager)
     {
         $logger->info('{user} is voting on answer {answer}!', [
-            'user' => $this->getUser()->getUserIdentifier(),
+            'user' => $this->getUser()->getEmail(),
             'answer' => $answer->getId(),
         ]);
 
