diff --git a/templates/question/homepage.html.twig b/templates/question/homepage.html.twig
index b04b8653..418fcf0f 100644
--- a/templates/question/homepage.html.twig
+++ b/templates/question/homepage.html.twig
@@ -24,6 +24,9 @@
                             <div class="vote-arrows vote-arrows-alt flex-fill pt-2" style="min-width: 90px;">
                                 <span>{{ question.votesString}} votes</span>
                             </div>
+                            {% for tag in question.tags %}
+                                <span class="badge rounded-pill bg-light text-dark">{{ tag.name }}</span>
+                            {% endfor %}
                         </div>
                         <div class="col">
                             <a class="q-title" href="{{ path('app_question_show', { slug: question.slug }) }}"><h2>{{ question.name }}</h2></a>
