diff --git a/.env b/.env
index 2cb271b..9501a4d 100644
--- a/.env
+++ b/.env
@@ -19,7 +19,3 @@ APP_SECRET=c28f3d37eba278748f3c0427b313e86a
 #TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
 #TRUSTED_HOSTS='^(localhost|example\.com)$'
 ###< symfony/framework-bundle ###
-
-###> sentry/sentry-symfony ###
-SENTRY_DSN=
-###< sentry/sentry-symfony ###
diff --git a/.env.local b/.env.local
index acd984d..e69de29 100644
--- a/.env.local
+++ b/.env.local
@@ -1 +0,0 @@
-SENTRY_DSN=https://7f45741877f3498eab0ae2bee6463d57@o372370.ingest.sentry.io/5186941
