diff --git a/src/Controller/RegistrationController.php b/src/Controller/RegistrationController.php
index b5bd7fd..186daef 100644
--- a/src/Controller/RegistrationController.php
+++ b/src/Controller/RegistrationController.php
@@ -5,6 +5,7 @@ namespace App\Controller;
 use App\Entity\User;
 use App\Form\RegistrationFormType;
 use App\Repository\UserRepository;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
@@ -58,7 +59,7 @@ class RegistrationController extends AbstractController
     /**
      * @Route("/verify", name="app_verify_email")
      */
-    public function verifyUserEmail(Request $request, VerifyEmailHelperInterface $verifyEmailHelper, UserRepository $userRepository): Response
+    public function verifyUserEmail(Request $request, VerifyEmailHelperInterface $verifyEmailHelper, UserRepository $userRepository, EntityManagerInterface $entityManager): Response
     {
         $user = $userRepository->find($request->query->get('id'));
         if (!$user) {
@@ -77,6 +78,11 @@ class RegistrationController extends AbstractController
             return $this->redirectToRoute('app_register');
         }
 
-        dd('TODO');
+        $user->setIsVerified(true);
+        $entityManager->flush();
+
+        $this->addFlash('success', 'Account Verified! You can now log in.');
+
+        return $this->redirectToRoute('app_login');
     }
 }
