diff --git a/src/Factory/UserFactory.php b/src/Factory/UserFactory.php
index b7b8ef99..72557247 100644
--- a/src/Factory/UserFactory.php
+++ b/src/Factory/UserFactory.php
@@ -44,6 +44,7 @@ final class UserFactory extends ModelFactory
             'email' => self::faker()->email(),
             'firstName' => self::faker()->firstName(),
             'plainPassword' => 'tada',
+            'isVerified' => true,
         ];
     }
 
