diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index fc13eef..6f8b274 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -36,7 +36,7 @@
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
                                 <p class="d-inline">{{ question.question|parse_markdown }}</p>
-                                <p class="pt-4"><strong>--{{ question.owner.displayName }}</strong></p>
+                                <p class="pt-4"><strong>--{{ question.owner.displayName }} ({{ question.owner.email }})</strong></p>
                             </div>
                         </div>
                     </div>
