diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 1f47ec69..38ef9104 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -97,7 +97,7 @@ EOF
     /**
      * @Route("/questions/{slug}/vote", name="app_question_vote", methods="POST")
      */
-    public function questionVote(Question $question, Request $request)
+    public function questionVote(Question $question, Request $request, EntityManagerInterface $entityManager)
     {
         $direction = $request->request->get('direction');
 
@@ -106,6 +106,11 @@ EOF
         } elseif ($direction === 'down') {
             $question->downVote();
         }
-        dd($question);
+
+        $entityManager->flush();
+
+        return $this->redirectToRoute('app_question_show', [
+            'slug' => $question->getSlug()
+        ]);
     }
 }
