diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index b3ad854d..f7d345dd 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -72,10 +72,7 @@ class QuestionController extends AbstractController
      */
     public function edit(Question $question)
     {
-        $this->denyAccessUnlessGranted('ROLE_USER');
-        if ($question->getOwner() !== $this->getUser()) {
-            throw $this->createAccessDeniedException('You are not the owner!');
-        }
+        $this->denyAccessUnlessGranted('EDIT', $question);
 
         return $this->render('question/edit.html.twig', [
             'question' => $question,
