diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index e1f1814..35ac5bc 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -99,6 +99,13 @@ EOF
      */
     public function questionVote(Question $question, Request $request)
     {
-        dd($question, $request->request->all());
+        $direction = $request->request->get('direction');
+
+        if ($direction === 'up') {
+            $question->setVotes($question->getVotes() + 1);
+        } elseif ($direction === 'down') {
+            $question->setVotes($question->getVotes() - 1);
+        }
+        dd($question);
     }
 }
