diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index b90d6c8..e512ff2 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -35,9 +35,11 @@
                             <div class="d-flex justify-content-between">
                                 <h1 class="q-title-show">{{ question.name }}</h1>
 
-                                <a href="{{ path('app_question_edit', {
-                                    slug: question.slug
-                                }) }}" class="btn btn-secondary btn-sm mb-2">Edit</a>
+                                {% if is_granted('EDIT', question) %}
+                                    <a href="{{ path('app_question_edit', {
+                                        slug: question.slug
+                                    }) }}" class="btn btn-secondary btn-sm mb-2">Edit</a>
+                                {% endif %}
                             </div>
                             <div class="q-display p-3">
                                 <i class="fa fa-quote-left mr-3"></i>
