diff --git a/migrations/Version20211012184326.php b/migrations/Version20211012184326.php
new file mode 100644
index 0000000..dbd1c82
--- /dev/null
+++ b/migrations/Version20211012184326.php
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20211012184326 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question ADD owner_id INT NOT NULL');
+        $this->addSql('ALTER TABLE question ADD CONSTRAINT FK_B6F7494E7E3C61F9 FOREIGN KEY (owner_id) REFERENCES user (id)');
+        $this->addSql('CREATE INDEX IDX_B6F7494E7E3C61F9 ON question (owner_id)');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question DROP FOREIGN KEY FK_B6F7494E7E3C61F9');
+        $this->addSql('DROP INDEX IDX_B6F7494E7E3C61F9 ON question');
+        $this->addSql('ALTER TABLE question DROP owner_id');
+    }
+}
