diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 38ef910..a45044a 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -109,6 +109,8 @@ EOF
 
         $entityManager->flush();
 
+        $this->addFlash('success', 'Vote tallied!');
+
         return $this->redirectToRoute('app_question_show', [
             'slug' => $question->getSlug()
         ]);
diff --git a/templates/base.html.twig b/templates/base.html.twig
index d73bdb9..dbe062c 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -18,6 +18,12 @@
           <button class="btn btn-dark">Sign up</button>
         </nav>
 
+        {% for flash in app.getFlashes('success') %}
+        <div class="alert alert-success">
+            <span class="fa fa-check"></span> {{ flash }}
+        </div>
+        {% endfor %}
+
         {% block body %}{% endblock %}
         <footer class="mt-5 p-3 text-center">
             Made with <i style="color: red;" class="fa fa-heart"></i> by the guys and gals at <a style="color: #444; text-decoration: underline;" href="https://symfonycasts.com">SymfonyCasts</a>
