diff --git a/migrations/Version20200708195925.php b/migrations/Version20200708195925.php
new file mode 100644
index 0000000..ea89e13
--- /dev/null
+++ b/migrations/Version20200708195925.php
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200708195925 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question ADD votes INT NOT NULL');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE question DROP votes');
+    }
+}
diff --git a/src/Entity/Question.php b/src/Entity/Question.php
index cbff45b..6686485 100644
--- a/src/Entity/Question.php
+++ b/src/Entity/Question.php
@@ -37,6 +37,11 @@ class Question
      */
     private $askedAt;
 
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $votes;
+
     public function getId(): ?int
     {
         return $this->id;
@@ -89,4 +94,16 @@ class Question
 
         return $this;
     }
+
+    public function getVotes(): ?int
+    {
+        return $this->votes;
+    }
+
+    public function setVotes(int $votes): self
+    {
+        $this->votes = $votes;
+
+        return $this;
+    }
 }
