diff --git a/templates/question/homepage.html.twig b/templates/question/homepage.html.twig
index 8f163042..3bc44f18 100644
--- a/templates/question/homepage.html.twig
+++ b/templates/question/homepage.html.twig
@@ -22,7 +22,7 @@
                         <div class="col-2 text-center">
                             <img src="{{ asset('images/tisha.png') }}" width="100" height="100"  alt="Tisha avatar">
                             <div class="vote-arrows vote-arrows-alt flex-fill pt-2" style="min-width: 90px;">
-                                <span>+ 6 votes</span>
+                                <span>{{ question.votesString}} votes</span>
                             </div>
                         </div>
                         <div class="col">
