diff --git a/src/Controller/QuestionController.php b/src/Controller/QuestionController.php
index 47394db8..dabed654 100644
--- a/src/Controller/QuestionController.php
+++ b/src/Controller/QuestionController.php
@@ -60,6 +60,8 @@ EOF
             $question->setAskedAt(new \DateTime(sprintf('-%d days', rand(1, 100))));
         }
 
+        $question->setVotes(rand(-20, 50));
+
         $entityManager->persist($question);
         $entityManager->flush();
 
