diff --git a/templates/question/show.html.twig b/templates/question/show.html.twig
index f9988432..643cbdbb 100644
--- a/templates/question/show.html.twig
+++ b/templates/question/show.html.twig
@@ -25,7 +25,7 @@
                                 <div class="vote-arrows vote-arrows-alt flex-fill pt-2" style="min-width: 90px;">
                                     <button class="vote-up btn btn-link" name="direction" value="up"><i class="far fa-arrow-alt-circle-up"></i></button>
                                     <button class="vote-down btn btn-link" name="direction" value="down"><i class="far fa-arrow-alt-circle-down"></i></button>
-                                    <span>+ 6</span>
+                                    <span>{{ question.votes }}</span>
                                 </div>
                             </div>
                         </div>
